types/bleno/index.d.ts

```diff
@@ -0,0 +1,128 @@
+// Type definitions for bleno 0.4
+// Project: https://github.com/sandeepmistry/bleno
+// Definitions by: Manuel Francisco Naranjo <naranjo.manuel@gmail.com>
```

Comment:
Prefer to put a GitHub handle here, like `<https://github.com/manuelnaranjo>`, so we can automatically notify you about new issues / pull requests about `bleno`.